e2e/rbd.go

```diff
@@ -510,6 +552,110 @@ var _ = Describe("RBD", func() {
 				// pvc clone is only supported from v1.16+
 				if k8sVersionGreaterEquals(f.ClientSet, 1, 16) {
 					validatePVCClone(pvcPath, pvcSmartClonePath, appSmartClonePath, f)
+					var wg sync.WaitGroup
```

Comment:
validatePVCClone already doing the validation by creating many clones, you need to add the data validation check to the `validatePVCClone` function. you need to do this check for Filesystem PVC we can add tests for BlockMode later.

Reply:
okay sure.